egs/wsj/s5/steps/diagnostic/analyze_lattice_depth_stats.py

```diff
@@ -29,13 +38,13 @@
 # set up phone_int2text to map from phone to printed form.
 phone_int2text = {}
 try:
-    f = open(args.lang + "/phones.txt", "r");
+    f = open(args.lang + "/phones.txt", "r", encoding='utf-8')
```

Comment:
this wouldn't work under python2, I believe (it does not have the encoding= option

Reply:
Hi Yenda, thanks for having a look!
You are right that the built-in open() doesn't have an "encoding" option. That's what the "from io import open" line is- i.e. we are re-binding the 'open' variable to point to a function that has this option and works for both Python2 and Python3. I've tested the scripts w/ Python versions 2.7 and 3.6.